xml/cha_administration.xml

```diff
@@ -584,4 +584,119 @@
+        </step>
+        <step>
+          <para>
+            Make sure your system is registered with a subscription that is eligible for LTSS. If
+            the system is not yet registered, register it
+            (see <xref linkend="sec-admin-register"/>).
+          </para>
+        </step>
+        <step>
+          <para>
+            Make sure the LTSS extension is available for your system:
+          </para>
+          <screen>&prompt.sudo;<command>SUSEConnect --list-extensions | grep LTSS</command>
+            SUSE Linux Enterprise Server LTSS &productnumber; x86_64
+            Activate with: SUSEConnect -p SLES-LTSS/&productnumber-regurl;/x86_64 -r ADDITIONAL REGCODE</screen>
+        </step>
+        <step>
+          <para>
+            Activate the module as instructed:
+          </para>
+          <screen>&prompt.sudo;<command>SUSEConnect -p SLES-LTSS/<replaceable>&productnumber-regurl;</replaceable>/<replaceable>x86_64</replaceable> -r <replaceable>REGISTRATION_CODE</replaceable></command></screen>
+        </step>
+      </procedure>
+    </sect2>
+    <sect2 xml:id="ltss-payg">
+      <title>LTSS on PAYG</title>
+      <para>
+        LTSS subscriptions for PAYG can be transacted through a private offer on the CSPs market
+        place or via direct transaction with SUSE. Reach out to <link
+        xlink:href="mailto:cloudsales@suse.com">cloudsales@suse.com</link> to work out the
+        commercial details.  You will receive a subscription and access to the <link
+          xlink:href="&sccurl;">&scc;</link>). With your subscription, you can activate a
+        registration code for LTSS.
+      </para>
+      <tip>
+        <title>Existing LTSS subscriptions</title>
+        <para>
+          If you already have an LTSS subscription that you are using in your data center, it will
+          work in the cloud just fine. You can deregister a system in your data center and move that
+          use of your LTSS subscription to an instance in the cloud.
+        </para>
+      </tip>
+      <procedure>
+        <para>
+          To enable the LTSS extension, perform the following steps:
+        </para>
+        <step>
+          <para>
+            Log in to the <link xlink:href="&sccurl;">&scc;</link> to activate a registration code.
+            Note that LTSS subscriptions are version-specific. If you have a subscription for
+            &slsa;&nbsp;15&nbsp;SP4, you cannot use that registration code to register LTSS on a
+            15&nbsp;SP3&nbsp;image. <emphasis>Make sure to activate the LTSS registration code for
+            the correct version and service pack (SP) of your instance!</emphasis>
+          </para>
+        </step>
+        <step>
+          <para>
+            Log in to your instance and make sure your system is registered with a subscription that
+            is eligible for LTSS. If the system is not yet registered, register it
+            (see <xref linkend="sec-admin-register"/>).
```

Comment:
Should we not tell users how to check that there instance is registered? And should we not tell users that for PAYG a registered system is the default state?

Since there is a difference between PAYG and BYOS I think we should not have the exact same wording.